prisma/schema.prisma

```diff
@@ -266,6 +268,26 @@ model Seeded {
   @@ignore
 }
 
+enum SwishOrderStatus {
+  PAID
+  DECLINED
+  ERROR
+  CANCELLED
+}
+
+model SwishOrder {
+  ID              Int               @id @default(autoincrement())
+  KID             String            @unique @db.VarChar(15)
+  instructionUUID String            @unique @db.VarChar(36) /// Swish uses this to identify the order
+  donorID         Int
+  donationID      Int?
+  status          SwishOrderStatus? /// External status from swish
```

Comment:
Do we need to allow null here? Haven't absorbed the details of the flow yet but guessing null would represent the order not being in either of the 4 current statuses. I'm thinking it might be clearer to add a fifth status in that case (`INITATED` maybe?) and set that as default

Reply:
Yes, should do something about this. The swish documentation is quite vague, the 4 statuses are the final result of any payment request. Looking into this!

Reply:
I made some changes here now. The idea is that it is null until a status from Swish has been retrieved. After a payment request is initiated, the client will poll this backend which will check with swish for payment status and persist it in our database as well.